metacat-common-server/src/main/java/com/netflix/metacat/common/server/properties/PropertyUtils.java

```diff
@@ -62,4 +64,26 @@ protected PropertyUtils() {
             return Lists.newArrayList();
         }
     }
+
+    /**
+     * Convert a delimited string into a Set of {@code QualifiedName}.
+     *
+     * @param names     The list of names to split
+     * @param delimiter The delimiter to use for splitting
+     * @return The list of qualified names
+     */
+    static Set<QualifiedName> delimitedStringsToQualifiedNamesSet(
+        @Nonnull @NonNull final String names,
```

Comment:
I thought we decided to flip to the other way, i.e. making Nonnull as default

Reply:
Kept it consistent with the other method in the class. We should do the massive change for all classes.